waku/node/waku_node.nim

```diff
@@ -898,16 +894,13 @@ proc stop*(node: WakuNode) {.async.} =
   await node.switch.stop()
   node.peerManager.stop()
 
-  when defined(rln):
-    if not node.wakuRlnRelay.isNil():
-      await node.wakuRlnRelay.stop()
+  if not node.wakuRlnRelay.isNil():
+    await node.wakuRlnRelay.stop()
 
   node.started = false
 
 proc isReady*(node: WakuNode): Future[bool] {.async.} =
-  when defined(rln):
-    if node.wakuRlnRelay == nil:
-      return false
-    return await node.wakuRlnRelay.isReady()
+  if node.wakuRlnRelay == nil:
```

Comment:
mm with this a node that doesnt have rln enabled will never be ready. perhaps we should add the `return true` back?
but yeah, problem is, if `wakuRlnRelay=nil` is it because i) rln is not enabled and we are ready or because ii) rln is enabled but not mounted and in this case we are not ready.

Reply:
wakuRlnRelay is nil whenever the `--rln-relay` flag is not passed in

Reply:
Addressed in 88bfe6ea